testproject/Packages/manifest.json

```diff
@@ -15,7 +15,6 @@
     "com.unity.test-framework.performance": "2.8.0-preview",
     "com.unity.textmeshpro": "3.0.6",
     "com.unity.timeline": "1.6.2",
-    "com.unity.transport": "1.0.0-pre.4",
```

Comment:
Does this mean that all this time, the test project tests were using UTP pre.4?

Reply:
no, PackageManager would pick the greatest meeting dependency version.
(see more there from the internal docs: https://confluence.unity3d.com/display/PAK/Dependency+Resolution)

Reply:
No, the dependency in `com.unity.netcode.gameobjects` would have installed the proper version.